src/ua/data_types/date_time.rs

```diff
@@ -18,6 +18,7 @@ impl DateTime {
 
 #[cfg(feature = "time")]
 impl TryFrom<time::OffsetDateTime> for DateTime {
+    // Explicit module path to avoid linter errors when feature is not enable by `#[cfg()]`.
```

Comment:
Do we need to mention this explicitly? This is a common linter issue and workaround in code behind feature flags. Commenting every usage of an explicit type specifier is unhandy and clutters the code.

Reply:
The only issue is that it becomes inconsistent if we omit these kinds of comments in new code. And I for my part won't repeat these comments ;)